scripts/bin/GenerateStaticData.pm

```diff
@@ -539,7 +537,6 @@ sub _texts {
         push @texts, localize('Pending');
         push @texts, localize('The ICO is currently unavailable.');
         push @texts, localize('Thank you for participating in our ICO. The final price of the tokens has been set at USD X.XX per token. Investors must deposit the balance owed on each successful bid based on the final price by 8 January 2018. You can proceed to claim the tokens with no remaining balance.');
-        push @texts, localize('Please select a value for account_opening_reason.');
         push @texts, localize('Minimum bid');
         push @texts, localize('The auction has ended. The final price of the tokens will be announced soon. Investors must deposit the balance owed on each successful bid based on the final price by 8 January 2018.');
```

Comment:
Would you please also update this message according to the latest changes have been done [here](https://github.com/binary-com/binary-static/pull/3305/files#diff-2fb0ba0560ac5636aa9fbb9ad0c589b0R41) and [here](https://github.com/binary-com/binary-static/pull/3306/files#diff-4497f6202765efc14ae17ee060dc1055R63)?
Thanks.